### PR TITLE
Bunch of fixes and enhancements

### PR DESCRIPTION
**Pull request by g-bougard**
This is the result of my first review. I'll go deeper in each category in another PR if necessary.

### CODE REVIEW

File: inventory.schema.json [hunk 1]
```diff
@@ -24,9 +48,11 @@
           ],
           "properties": {
             "logdate": {
-              "type": "string",
-              "format": "date-time"
-            },
+              "examples": [
```
Comment (trasher) [inline inventory.schema.json]: Example here is maybe not mandatory, it's provided from definition

**Comment by trasher** [comment]
Looks good to me, thank you :)

File: inventory.schema.json (hunk 1 above)
Comment (g-bougard) [reply inventory.schema.json]: I agree it's not mandatory, but I prefer to see examples next to the parameter definition when I check the specs. And in the case someone decides to update the definition this permits to double-check the new definition still applies here
This is just a point of view.  ;-)

Comment (trasher) [reply inventory.schema.json]: OK, no problem

**Pull request merged**